src/App.js

```diff
@@ -78,22 +79,7 @@ class App extends Component {
                             content={PageContent.CALENDAR}>
                         </PageContainer>
                     :
-                        /* Show landing page if user is logged out */
-
-                        /* ============= Temp Testing Code ============= */
-                        /* TODO: 
-                         * 1. Replace the temp debug signin with SignUpIn page 
-                         * 2. Remove tempSignIn function
-                         */
```

Comment:
Here's the TODO comment for when you replace this temporary code with your real sign in/up code. So please also remove the block of code for tempSignIn() above (in between line 46-68)